packages/flutter_tools/lib/src/commands/migrate.dart

```diff
@@ -2,35 +2,35 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// import 'package:process/process.dart';
+import 'package:process/process.dart';
 
-// import '../base/file_system.dart';
+import '../base/file_system.dart';
 import '../base/logger.dart';
-// import '../base/platform.dart';
+import '../base/platform.dart';
 import '../base/terminal.dart';
 import '../migrate/migrate_utils.dart';
 import '../runner/flutter_command.dart';
 // TODO(garyq): Add each of these back in as they land.
 // import 'migrate_abandon.dart';
```

Comment:
Ignore the commented lines, this will all need to be updated/cleaned up depending on the order these subcommands land. These commented lines won't make it into the codebase.